docs.json

```diff
@@ -1,15 +1,31 @@
 {
   "name": "React Native Google Mobile Ads",
+  "logo": "https://raw.githubusercontent.com/DoctorJohn/react-native-google-ads/main/docs/img/logo_admob_192px.svg",
```

Comment:
We have this logo already in the repo and we should not use content directly from a user.
https://github.com/invertase/react-native-google-mobile-ads/blob/main/docs/img/logo_admob_192px.svg

Reply:
Good catch! No idea how I mistakingly ended up pasting the URL from my fork :eyes: I fixed it :+1: